termine-fe/src/App.js

```diff
@@ -229,13 +234,13 @@ function App() {
             {window.config.longInstanceName}
           </h3>
           <Link className="button" to={TAB.BOOK}>
-            Termine buchen
+            <Trans>Book appointments</Trans>
           </Link>
           <Link className="button" to={TAB.BOOKED}>
-            Meine Buchungen
+            <Trans>My Appointments</Trans>
```

Comment:
Inconsistent  capitalisation

Reply:
removed capitalisation of appointments, good catch!